cost-analyzer/templates/_helpers.tpl

```diff
@@ -175,6 +216,14 @@ app: query-service
 {{ include "kubecost.federator.chartLabels" . }}
 app: federator
 {{- end -}}
+{{- define "aggregator.commonLabels" -}}
+{{ include "kubecost.aggregator.chartLabels" . }}
+app: kubecost-aggregator
```

Comment:
Should this be `cost-analyzer.chartLabels`? And do we really want to specify the `app` label here? I think it is also set in individual manifest definitions.

Reply:
fixed it to reference cost-analyzer. if anything, I would vote to refactor out the app: from all the manifests and keep this here. another day perhaps? :)